projects/vdk-core/plugins/vdk-greenplum/tests/test_vdk_greenplum_utils.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+import unittest
+from unittest import mock
+
+import pytest
+from click.testing import Result
+from vdk.plugin.greenplum import greenplum_plugin
+from vdk.plugin.test_utils.util_funcs import cli_assert_equal
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
+
+VDK_DB_DEFAULT_TYPE = "VDK_DB_DEFAULT_TYPE"
+VDK_GREENPLUM_DBNAME = "VDK_GREENPLUM_DBNAME"
+VDK_GREENPLUM_USER = "VDK_GREENPLUM_USER"
+VDK_GREENPLUM_PASSWORD = "VDK_GREENPLUM_PASSWORD"
+VDK_GREENPLUM_HOST = "VDK_GREENPLUM_HOST"
+VDK_GREENPLUM_PORT = "VDK_GREENPLUM_PORT"
+
+
+@pytest.mark.usefixtures("greenplum_service")
+@mock.patch.dict(
+    os.environ,
+    {
+        VDK_DB_DEFAULT_TYPE: "GREENPLUM",
+        VDK_GREENPLUM_DBNAME: "postgres",
```

Comment:
This seems weird, unless it's some joke I'm not getting.

Reply:
Greenplum is PosgtreSQL-based :) using same `postgres` default schema